package.json

```
@@ -4,18 +4,21 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "build": "babel src --out-dir dist",
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "db:roll": "sequelize db:migrate",
```

`db:roll`??? Please change to `db:migrate`. Also, instead of using babelHook which will need us to move @babel/register to dependencies instead of devDependencies, I believe we should use my solution i.e
```suggestion
    "sequelize-cli:es6": "babel-node ./node_modules/.bin/sequelize $*"
```
This uses babel-node to run sequelize therefore transpiling it on each run.

Then the following scripts will be upgraded to

```
    "db:migrate": "npm run sequelize-cli:es6 db:migrate",
    "db:undo": "npm run sequelize-cli:es6 db:migrate:undo:all",
    "db:setup": "npm run db:migrate && npm run sequelize-cli:es6 db:seed:all",
```

In the config, we'll now have

```
'config': path.resolve('./config/sequelize'),
```

---

I see your point now